app/models/ordergroup.rb

```diff
@@ -148,6 +148,26 @@ def account_updated
     financial_transactions.last.try(:created_on) || created_on
   end
 
+  def self.sort_by_param(param)
+    param ||= "name"
```

Comment:
i personally would prefer `sort_by_param(param = "name")`, since it's IMHO clearer

Reply:
That was also my first idea. But it turned out this only works when the parameter is not used, but not when NIL is passed.
Let me know if I might have missed something?